dipy/io/utils.py

```diff
@@ -91,3 +91,42 @@ def decfa(img_orig, scale=False):
     new_hdr.set_data_dtype(dest_dtype)
 
     return Nifti1Image(out_data, affine=img_orig.affine, header=new_hdr)
+
+
+def decfa_to_float(img_orig):
+    """
+    Create a nifti-compliant directional-encoded color FA file.
+
+    Parameters
+    ----------
+    img_orig : Nifti1Image class instance.
+        Contains encoding of the DEC FA image with a 3D volume of data, where
+        each element is a (R, G, B) tuple in uint8.
+
+    Returns
+    -------
+    img : Nifti1Image class instance with uint8 dtype.
```

Comment:
Written that it's going to be in `uint8`, but you convert to `np.float` in the header and the name of the function says so.